src/official-account/official-account.ts

```diff
@@ -1,42 +1,46 @@
 /* eslint-disable camelcase */
 import { EventEmitter } from 'events'
+import { v4 } from 'uuid'
 
 import crypto from 'crypto'
-import { ContactGender, FileBox, log }  from 'wechaty-puppet'
+import { ContactGender, FileBox, log } from 'wechaty-puppet'
 import { FileBoxType } from 'file-box'
 
 import { normalizeFileBox } from './normalize-file-box'
 
 import {
   Webhook,
   VerifyArgs,
-}                      from './webhook'
+} from './webhook'
 import { PayloadStore } from './payload-store'
 import {
   getSimpleUnirest,
   SimpleUnirest,
-}                       from './simple-unirest'
+} from './simple-unirest'
 import {
   OAMessageType,
   // OAMediaPayload,
   OAMediaType,
   ErrorPayload,
   OAContactPayload,
-}                       from './schema'
+  OATagPayload,
+  OAMessagePayload,
+} from './schema'
+import { getTimeStampString } from './utils'
 
 export interface OfficialAccountOptions {
-  appId            : string,
-  appSecret        : string,
-  port?            : number,
-  token            : string,
-  webhookProxyUrl? : string,
-  personalMode?    : boolean,
+  appId: string,
```

Comment:
Please keep the spaces for keeping the formatting style consistent.

Reply:
Ok, I have ignored these code style, I will change it later.